.drone/drone.yml

```diff
@@ -228,7 +228,7 @@ steps:
       GITHUB_TOKEN:
         from_secret: GITHUB_KEY
     commands:
-      - apt-get install -y apt-transport-https ca-certificates curl gnupg lsb-release
+      - apt-get update &&  apt-get install -y rubygems rpm nsis apt-transport-https ca-certificates curl gnupg lsb-release libbpfcc-dev
```

Comment:
Why didn't this break before? 🤔

Reply:
I have no idea. Which is a bit scary.

Reply:
Oh, I bet it's just `libbpfcc-dev`, which wasn't needed prior to v0.26.0 since that's when the eBPF integration was introduced. The previous install step probably was a no-op, but I think the seego based image was already installing those anyway.